src/components/pill-selector-dropdown/PillSelector.scss

```diff
@@ -91,8 +91,7 @@
             }
         }
 
-        .pill-selector-input,
-        input[type='text'].pill-selector-input {
```

Comment:
Couldn't find a reason why `input[type='text']` was added here, looks like `.pill-selector-input` should've been enough